src/contracts/SwapPair.cdc

```diff
@@ -5,68 +5,97 @@
 # Author: Increment Labs
 
 */
-import FungibleToken from "./tokens/FungibleToken.cdc"
+import FungibleToken from "./env/FungibleToken.cdc"
 import SwapInterfaces from "./SwapInterfaces.cdc"
 import SwapConfig from "./SwapConfig.cdc"
 import SwapError from "./SwapError.cdc"
 import SwapFactory from "./SwapFactory.cdc"
 
-pub contract SwapPair: FungibleToken {
+access(all) contract SwapPair: FungibleToken {
     /// Total supply of pair lpTokens in existence
-    pub var totalSupply: UFix64
+    access(all) var totalSupply: UFix64
     
     /// Two vaults for the trading pair.
-    access(self) let token0Vault: @FungibleToken.Vault
-    access(self) let token1Vault: @FungibleToken.Vault
-    pub let token0VaultType: Type
-    pub let token1VaultType: Type
-    pub let token0Key: String
-    pub let token1Key: String
+    access(self) let token0Vault: @{FungibleToken.Vault}
+    access(self) let token1Vault: @{FungibleToken.Vault}
+    access(all) let token0VaultType: Type
+    access(all) let token1VaultType: Type
+    access(all) let token0Key: String
+    access(all) let token1Key: String
     
     /// TWAP: last cumulative price
-    pub var blockTimestampLast: UFix64
-    pub var price0CumulativeLastScaled: UInt256
-    pub var price1CumulativeLastScaled: UInt256
+    access(all) var blockTimestampLast: UFix64
+    access(all) var price0CumulativeLastScaled: UInt256
+    access(all) var price1CumulativeLastScaled: UInt256
 
     /// Transaction lock 
     access(self) var lock: Bool
     
     /// √(reserve0 * reserve1) for volatile pool, or √√[(r0^3 * r1 + r0 * r1^3) / 2] for stable pool, as of immediately after the most recent liquidity event
-    pub var rootKLast: UFix64
+    access(all) var rootKLast: UFix64
 
     /// Reserved parameter fields: {ParamName: Value}
     /// Used fields:
     ///   |__ 1. "isStableSwap" -> Bool
     access(self) let _reservedFields: {String: AnyStruct}
 
     /// Event that is emitted when the contract is created
-    pub event TokensInitialized(initialSupply: UFix64)
+    access(all) event TokensInitialized(initialSupply: UFix64)
     /// Event that is emitted when tokens are withdrawn from a Vault
-    pub event TokensWithdrawn(amount: UFix64, from: Address?)
+    access(all) event TokensWithdrawn(amount: UFix64, from: Address?)
     /// Event that is emitted when tokens are deposited to a Vault
-    pub event TokensDeposited(amount: UFix64, to: Address?)
+    access(all) event TokensDeposited(amount: UFix64, to: Address?)
```

Comment:
I'd recommend removing these and relying on the default `FungibleToken` events instead: https://github.com/onflow/flow-ft/blob/v2-standard/contracts/FungibleToken.cdc#L46-L50